tools/defaults.bzl

```diff
@@ -35,6 +31,25 @@ PKG_GROUP_REPLACEMENTS = {
     "RXJS_VERSION": RXJS_VERSION,
 }
 
+# Convert a package name on npm to an identifier that's a legal global id
+# @ngrx/store -> ngrx.store
+# @ngrx/router-store -> ngrx.routerStore
+def _global_name(package_name):
+    # strip npm scoped package qualifier
+    start = 1 if package_name.startswith("@") else 0
+    parts = package_name[start:].split("/")
+    result_parts = []
+    for p in parts:
+        # convert dash case to camelCase
+        # First letter in the result is always unchanged
+        result = p[0] + "".join([p2.title() for p2 in p.split("-")])[1:]
+        result_parts.append(result)
+    return ".".join(result_parts)
+
+NGRX_GLOBALS = dict({
+    "tslib": "tslib",
```

Comment:
@gregmagolan what do you reckon if for version 9 we move the global name logic in https://github.com/angular/angular/blob/8c76e78ec454b0b9c2d1fadb6c3d1d1202772ea5/packages/bazel/src/ng_package/ng_package.bzl#L223